persistence-api/src/main/java/io/stargate/db/schema/Column.java

```diff
@@ -359,7 +358,7 @@ public boolean isSet() {
 
     Smallint(19, Short.class, false, "16-bit signed integer"),
 
-    Text(10, String.class, true, "UTF-8 encoded string"),
+    Text(13, String.class, true, "UTF-8 encoded string"),
```

Comment:
Can this have some kind of backward compatibility issue? I guess not, but just to make sure?

Reply:
I think it's fairly safe given that code 10 doesn't exist in the protocol. It's not possible for a client to send it -- nor accept it -- unless it was buggy.